src/Common/src/Interop/Windows/kernelbase/Interop.OpenCommPort.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Win32.SafeHandles;
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+
+internal partial class Interop
+{
+    internal partial class KernelBase
+    {
+        [DllImport(Libraries.KernelBase, SetLastError = true)]
```

Comment:
Just making sure, did you try running the tests on uapaot? I ask because sometimes we need to have the EntryPoint specified here.